Simplenote/src/main/java/com/automattic/simplenote/NoteListFragment.java

```diff
@@ -1529,38 +1531,34 @@ protected Void doInBackground(Void... args) {
 
         @Override
         protected void onPostExecute(Void aVoid) {
-            NoteListFragment fragment = fragmentRef.get();
-
+            NoteListFragment fragment = mNoteListFragmentReference.get();
             fragment.mActionMode.finish();
             fragment.refreshList();
         }
     }
 
-    private static class trashNotesTask extends AsyncTask<Void, Void, Void> {
-
-        private SoftReference<NoteListFragment> fragmentRef;
+    private static class TrashNotesTask extends AsyncTask<Void, Void, Void> {
         private List<String> mDeletedNoteIds = new ArrayList<>();
+        private SoftReference<NoteListFragment> mNoteListFragmentReference;
         private SparseBooleanArray mSelectedRows = new SparseBooleanArray();
 
-        private trashNotesTask(NoteListFragment context) {
-            fragmentRef = new SoftReference<>(context);
+        private TrashNotesTask(NoteListFragment context) {
+            mNoteListFragmentReference = new SoftReference<>(context);
         }
 
         @Override
         protected void onPreExecute() {
-            NoteListFragment fragment = fragmentRef.get();
-            if (fragment.getListView() != null) {
-                mSelectedRows = fragment.getListView().getCheckedItemPositions();
-            }
+            NoteListFragment fragment = mNoteListFragmentReference.get();
+            mSelectedRows = fragment.getListView().getCheckedItemPositions();
```

Comment:
Is there a reason the null check is removed?

Reply:
It is unnecessary since `fragment.getListView()` is never `null`.